tests/test_package.py

```diff
@@ -127,19 +127,19 @@ def test_reset(postgres_db_engine: sa.Engine) -> None:
             cwd=temp_path,
         )
 
-        # Create a system database and verify it exists
-        subprocess.check_call(["dbos", "migrate"], cwd=temp_path)
+        # Call reset and verify it's destroyed
+        subprocess.check_call(["dbos", "reset", "-y"], cwd=temp_path)
         with postgres_db_engine.connect() as c:
             c.execution_options(isolation_level="AUTOCOMMIT")
             result = c.execute(
                 sa.text(
                     f"SELECT COUNT(*) FROM pg_database WHERE datname = '{sysdb_name}'"
                 )
             ).scalar()
-            assert result == 1
+            assert result == 0
 
-        # Call reset and verify it's destroyed
-        subprocess.check_call(["dbos", "reset", "-y"], cwd=temp_path)
+        # Call reset with a specific sys db name and verify it's destroyed
+        subprocess.check_call(["dbos", "reset", "-y", "-s", sysdb_name], cwd=temp_path)
```

Comment:
What is this test even testing?  We removed the step that creates a database, so we just destroy a non-existent database twice?

Reply:
reset will create the system database by default when it the client instantiates a new system DB object.

Reply:
That's a pure implementation detail, not something the test should rely on. The reset command doesn't even do anything with the system database it creates!